src/main/scala/edu/ie3/simona/model/participant/WecModel.scala

```diff
@@ -274,7 +277,7 @@ object WecModel {
       scaledInput.getId,
       operationInterval,
       QControl(scaledInput.getqCharacteristics),
-      Kilowatts(
+      Kilovoltamperes(
         scaledInput.getType.getsRated.to(KILOWATT).getValue.doubleValue
       ),
```

Comment:
We may have to think about psdm changes as well?

Reply:
I will change this to `KILOVOLTAMPERE`.

Reply:
Alright, thank you!